src/stylesheets/_variables.scss

```diff
@@ -28,4 +29,6 @@ $margin-xlarge: 25px;
 $margin-xxlarge: 30px;
 $margin-xxxlarge: 35px;
 
-$standard-form-element-height: 38px
+$margin-form: 80px;
+
+$standard-form-element-height: 36px
```

Comment:
See comment on semantic names above. You can reuse existing size variables where appropriate or create more granular ones that can be used here or other places.